scripts/feature_tests/pivoting/Makefile

```diff
@@ -15,3 +15,6 @@ repivoting:
 
 deprovision:
 	./../feature_test_deprovisioning.sh
+
+upgrading:
+	./upgrade.sh
```

Comment:
The upgrade tests are run by script:
https://github.com/metal3-io/metal3-dev-env/blob/master/scripts/feature_tests/upgrade/upgrade.sh
Why not use that directly after the pivoting is finished?

Reply:
In pivoting it also runs the repivoting after the pivoting is done. In our case we need to run the upgrade for 3cp+1w priority use case after pivoting. To call the pivoting with the master and worker node variable we need separate script to call the pivoting.